classes/observation_manager.php

```diff
@@ -296,4 +298,96 @@ public static function reorder_observation_point(int $observationid, int $obpoin
 
         $transaction->allow_commit();
     }
+
+    /**
+     * Returns a list of observation points with responses for a particular session
+     * @param int $observationid ID of the observation instance
+     * @param int $sessionid ID of the observation sesssion
+     * @return array array of observation points with responses for the given session
+     */
+    public static function get_points_and_responses(int $observationid, int $sessionid) {
+        global $DB;
+
+        // Selects all points for this observation,
+        // and attaches responses for the current session (if one exists).
+
+        $sql = 'SELECT pts.id as point_id, obs_id, title, list_order, ins, ins_f, max_grade, res_type,
```

Comment:
SQL like this is pretty hard to read + grok. There is a nice formatting standard here:

https://docs.moodle.org/dev/SQL_coding_style#Indentation

This makes it much easier to read. Would be nice to convert them over